tools/src/scie_pants/install_pants.py

```diff
@@ -120,23 +120,36 @@ def install_pants_from_pex(
                 " or file an issue on GitHub: https://github.com/pantsbuild/pants/issues/new/choose.\n\n"
                 f"Exception:\n\n{e}"
             )
-        subprocess.run(
-            args=[
-                sys.executable,
-                pants_pex.name,
-                "venv",
-                "--prompt",
-                prompt,
-                "--compile",
-                "--pip",
-                "--collisions-ok",
-                "--no-emit-warnings",  # Silence `PEXWarning: You asked for --pip ...`
-                "--disable-cache",
-                str(venv_dir),
-            ],
-            env={"PEX_TOOLS": "1"},
-            check=True,
-        )
+        try:
+            pants_venv_result = subprocess.run(
+                args=[
+                    sys.executable,
+                    pants_pex.name,
+                    "venv",
+                    "--prompt",
+                    prompt,
+                    "--compile",
+                    "--pip",
+                    "--collisions-ok",
+                    "--no-emit-warnings",  # Silence `PEXWarning: You asked for --pip ...`
+                    "--disable-cache",
+                    str(venv_dir),
+                ],
+                env={"PEX_TOOLS": "1"},
+                check=True,
+                capture_output=True,
+            )
+            with open(str(venv_dir / "pants-install.log"), "a") as fp:
```

Comment:
This currently incorporates both stdout and stderr into one file without separators and not with the content in chronological order. What do you think about either:
- writing to separate files 
- combining both streams into one so that the output is properly interleaved

>  If you wish to capture and combine both streams into one, use stdout=PIPE and stderr=STDOUT instead of capture_output. &mdash; https://docs.python.org/3/library/subprocess.html#subprocess.run

Reply:
Ah, good point. I think I'd prefer a single combined stream rather than two files. I'll push for that.